src/pages/LoanFinding/LoanFinding.vue

```diff
@@ -171,6 +169,7 @@ export default {
 				});
 
 				return Promise.all([
+					client.query({ query: userInfoQuery }),
```

Comment:
Is there a reason why the user info was queried first before?

Reply:
No reason... It was the first query implemented in the page. any suggestion about this? do you mind a reason to don't be first?